session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1230,6 +1231,38 @@ public void insertRecordsOfOneDevice(
     }
   }
 
+  /**
+   * Insert multiple rows, which can reduce the overhead of network. This method is just like jdbc
+   * executeBatch, we pack some insert request in batch and send them to server. If you want improve
+   * your performance, please see insertTablet method
+   *
+   * <p>Each row could have same deviceId but different time, number of measurements, number of
+   * values as String
+   *
+   * @param haveSorted deprecated, whether the times have been sorted
+   */
+  public void insertStringRecordsOfOneDevice(
```

Comment:
Please also add this interface at SessionPool.java

Reply:
I've added 4 interfaces(insertStringRecordsOfOneDevice with haveSorted, insertStringRecordsOfOneDevice, insertAlignedStringRecordsOfOneDevice with haveSorted, insertAlignedStringRecordsOfOneDevice) to SessionPool.java.